boutdata/data.py

```diff
@@ -1057,6 +1057,7 @@ def __init__(
         Initialise BoutOutputs object
         """
         self._path = path
+        self._file0 = None
         # normalize prefix by removing trailing '.' if present
         self._prefix = prefix.rstrip(".")
```

Comment:
`removesuffix` should be used:
```suggestion
        self._prefix = prefix.removesuffix(".")
```

Available since 3.9 - can we use that yet?
https://peps.python.org/pep-0616/